src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  width: 198px;
```

Comment:
better way to create reset file and set box-sizing: border box, then you dont need to calculate border and write so weird sizes

Reply:
https://github.com/mate-academy/layout_catalog/pull/3507/files#r1282205832

add box-sizing: border box in `*` selector